go/runtime/history/db.go

```diff
@@ -28,10 +28,9 @@ var (
 	//
 	// Value is CBOR-serialized roothash.AnnotatedBlock.
 	blockKeyFmt = keyFormat.New(0x02, uint64(0))
-	// roundResultsKeyFmt is the round result index key format.
-	//
-	// Value is CBOR-serialized roothash.RoundResults.
-	roundResultsKeyFmt = keyFormat.New(0x03, uint64(0))
+
+	// Do not use 0x03 prefix. Previously it was used for round results key.
+	// Old reindexes may still have legacy data stored.
```

Comment:
I would rather rename to `deprecatedRoundResultsKeyFmt = keyFormat.New(0x03, uint64(0))`, `reservedKeyFmt = keyFormat.New(0x03, uint64(0))`, or `deprecatedKeyFmt = keyFormat.New(0x03, uint64(0))` and update the comment, because the key format will panic if prefix is used twice, while your comment can easily be overlooked. 

See how some other vars are deprecated.

Reply:
Ah yes, this is even better.